mips-unknown-elf doesn't appear to like this

```
Error:     cc1: error: position-independent code requires '-mabicalls'
Error:     cc1: error: position-independent code requires '-mabicalls'
Error:     make[3]: *** [/home/runner/work/crosstool-ng/crosstool-ng/.build/mips-unknown-elf/src/gcc/gcc/c/Make-lang.in:128: s-selftest-c] Error 1
Error:     make[2]: *** [Makefile:4594: all-gcc] Error 2
Error:     make[1]: *** [Makefile:1014: all] Error 2
```

https://github.com/crosstool-ng/crosstool-ng/suites/7792031688/artifacts/329238278

--- a/config/cc/gcc.in
+++ b/config/cc/gcc.in
@@ -197,6 +197,17 @@ config CC_GCC_LTO_ZSTD
 #-----------------------------------------------------------------------------
 comment "Settings for libraries running on target"
 
+config CC_GCC_ENABLE_DEFAULT_PIE
+    bool
+    prompt "Enable Position Independent Executable as default"
+    default y
+    depends on GCC_6_or_later && ! WINDOWS && ! ARCH_MIPS
+    help
+      Pass --enable-default-pie to crossgcc's configure.
+      
+      This will let crossgcc compile executable as Position Independent
+      Executable by default.
+
 config CC_GCC_ENABLE_TARGET_OPTSPACE
     bool
     prompt "Optimize gcc libs for size"

--- a/scripts/build/cc/gcc.sh
+++ b/scripts/build/cc/gcc.sh
@@ -436,6 +436,10 @@ do_gcc_core_backend() {
         extra_config+=("--with-host-libstdcxx=${host_libstdcxx_flags[*]}")
     fi
 
+    if [ "${CT_CC_GCC_ENABLE_DEFAULT_PIE}" = "y" ]; then
+        extra_config+=("--enable-default-pie")
+    fi
+
     if [ "${CT_CC_GCC_ENABLE_TARGET_OPTSPACE}" = "y" ] || \
        [ "${enable_optspace}" = "yes" ]; then
         extra_config+=("--enable-target-optspace")

